static/src/javascripts/projects/common/modules/commercial/support-utilities.ts

```diff
@@ -0,0 +1,35 @@
+import {
+	countryCodeToSupportInternationalisationId,
+	getCountryCode,
+} from '../../../../lib/geolocation';
+
+type SupportUrlProtocol = 'https://';
+type SupportUrlDomain = 'support.theguardian.com';
+type SupportUrlPath = 'contribute' | 'subscribe';
+type SupportUrlRaw =
+	`${SupportUrlProtocol}${SupportUrlDomain}/${SupportUrlPath}`;
+
+const addCountryGroupToSupportLink = (rawUrl: SupportUrlRaw): string => {
+	const countryCode = getCountryCode();
+	const countryGroup =
+		countryCodeToSupportInternationalisationId(countryCode);
+	return rawUrl.replace(
+		/(support.theguardian.com)\/(contribute|subscribe)/,
```

Comment:
FYI, this will create a `new RegExp` every time it is called. A benefit of extracting regular expressions to a constant is that you can add 

```ts
/**
 * This captures the URLs containing the followin sequences:
 * - `support.theguardian.com/contribute`
 * - `support.theguardian.com/subscribe`
 *
 * This allows inserting a country group before the path.
 */
const REGEX = /(support.theguardian.com)\/(contribute|subscribe)/
```